src/containers/Body.module.scss

```diff
@@ -19,11 +21,24 @@
       background-color: $white;
     }
   }
+
+  @media (min-width: $mobile_breakpoint) {
+    flex-direction: row;
+    gap: 30px;
+  }
+
+  section {
+    .single_date {
+      margin-top: 20px;
+      border-radius: 10px;
+      width: auto;
+    }
+  }
 }
 
 .cards {
   display: flex;
   flex-direction: column;
   row-gap: 30px;

```

Comment:
Solo su mobile, setterei questo valore a 20, in modo che sia visivamente coerente col padding-top.